resdk/resolwe.py

```diff
@@ -156,8 +153,7 @@ def _register(self, src, slug):
             process['persistence'] = persistence_map[process['persistence']]
 
         try:
-            server_process = self.process.filter(slug=process['slug'],
-                                                 ordering='-version', limit=1)
+            server_process = self.process.filter(slug=process['slug'], ordering='-version')[:1]
```

Comment:
It's just a bit ugly to have `ordering` in `filter`. Maybe we can add `.order_by` method in the future. And also `.first()` and `.last()` would be useful.